charts/mgmt-agent/values.yaml

```diff
@@ -78,7 +85,13 @@ deployment:
    
    daemonSet:
       # Provide the host path if Agent is deployed as DaemonSet. Management Agent Pod should have read-write access to it.
+      # Note: The hostPath cannot point to /opt/oracle, please provide another location or create a sub-directory under /opt/oracle and use that as hostPath.
+      # The Management Agent helm chart will attempt to change ownership and permissions on the host path provided here.
```

Comment:
This comment says, ownership and permission will be modified for the path given here but the command in initContainer has hardcoded value to /opt/oracle. Either of these has to be corrected ?